### ISSUE TITLE
VS code extension pane view for "designer" and "Overview" is blank WSL2

### ISSUE THREAD

**Issue by Bandgren** [Azure/logicapps#231]
Hi,

Im running into some issue with the Logic apps (preview) extension when im trying to open either the `designer` or `overview` views in VS code.
Im running on WSL2 with Ubuntu. After reading some issues here for example https://github.com/Azure/logicapps/issues/201 so until Azurite or the logic apps team decide on anything else i had to create a Storage account in the Azure portal and set the 
`"AzureWebJobsStorage": "DefaultEndpointsProtocol=....` ENV in both of the `local.settings.json` files.

So the Logic apps start up fine but i when i try to open the overview or the designer view the pane is completely empty
![image](https://user-images.githubusercontent.com/53816703/104929293-fc28b680-59a3-11eb-92e6-de5501fd3869.png)

Running it "natively" on windows with the same storage account the overview and designer view panes are working as expected.

is this a known issue that i have missed somewhere?

**Comment by Bandgren** [comment]
@ecfan Could this be that the Logic apps (preview) extension is not playing nice when VS code is running the remote extension to either a devcontainer or to WSL2? 🤔

**Comment by ecfan** [comment]
Hi @Bandgren,

I'm including the dev folks to help answer. 

@rohithah and @laveeshb, any ideas why the designer and overview panes aren't opening?

**Comment by laveeshb** [comment]
@Bandgren -- can you trace network calls when you navigate to overview or designer view? We should check if any requests to fetch data to show on the designer are failing.

**Comment by Bandgren** [comment]
Hi, @laveeshb, @ecfan Thanks for the attention, i went ahead and made two network captures when i open the designer view. On a logic app preview project on windows we can see that all network calls works fine (image one). 
But on a logic app preview project running from WSL2 e.g via the remote extention in VS code we get a 404 when we the preview extention tries to call the following url: (image two)
```
http://172.27.91.56:42129/vscode-remote-resource?tkn=ef79fd15-f9c8-40fc-bb05-706f2902c5e4&path=%2Fhome%2Fbandgren%2F.vscode-server%2Fextensions%2Fms-azuretools.vscode-azurelogicapps-0.0.7%2Fdist%2Fdesigner%2Fdraft.min.js
```

I have also tried to uninstall and reinstall the extentions multiple times in the wsl2 instance. 

Running the extention and project via a `devcontainer` i see the same 404 when the exntention is trying to GET the same uri but towards the devcontainer
```
http://127.0.0.1:36399/vscode-remote-resource?tkn=175de0be-6637-457b-b587-1f1ee370eb72&path=%2Froot%2F.vscode-server%2Fextensions%2Fms-azuretools.vscode-azurelogicapps-0.0.7%2Fdist%2Fdesigner%2Fdraft.min.js
```
## windows
![windows](https://user-images.githubusercontent.com/53816703/105829272-69e86a00-5fc4-11eb-9d12-9b9859a6d3dd.png)

## wsl2
![wsl2](https://user-images.githubusercontent.com/53816703/105829374-969c8180-5fc4-11eb-8fa7-c77682bb4c60.png)

WireShark PCAP files from the 2 captures.
[la_preview_designer_pcap_files.zip](https://github.com/Azure/logicapps/files/5872565/la_preview_designer_pcap_files.zip)

**Comment by laveeshb** [comment]
Thanks @Bandgren. We will have our team fix it for Linux.

**Comment by laveeshb** [comment]
@Bandgren We don't support the VS code based authoring tools on a Linux machine at this time but we do intend to add support for them in future. We have updated our docs to call it out -- https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-overview-preview#limited-unavailable-unsupported

**Comment by joechung-msft** [comment]
I merged the fix for the Draft script filename case. It will be included in the next extension release (0.0.9).

**Comment by joechung-msft** [comment]
0.0.9 is out now.
